run-poc/redefine-mst-parser.scm

```diff
@@ -106,30 +215,43 @@
 	(close-port file-port)
 )
 
-(define-public (observe-mst-mode plain-text CNT-MODE MST-DIST EXPORT-MST)
+(define-public (observe-mst-mode plain-text SENT-NBR CNT-MODE MST-DIST EXPORT-MST)
 "
   observe-mst-mode -- update pseduo-disjunct counts by observing raw text.
-                      Build mst-parses using MI calculated beforehand.
-                      When MST-DIST is true, word-pair MI values are adjusted
-                      for distance.
-                      Obtained parses are exported to file if EXPORT-MST
-                      is true.
+					  Build mst-parses using MI calculated beforehand.
+					  When MST-DIST is true, word-pair MI values are adjusted
+					  for distance.
+					  Obtained parses are exported to file if EXPORT-MST
+					  is true.
```

Comment:
this screws up printing of the documentation. So, the next user who tries `,d observe-mst-mode` will get this crazy indented thing.

Recall that in guile `,d` or `,describe` prints the subroutine documentation, and `,a` or `,apropos` prints the names of all related subroutines.

Reply:
Fixing it...